radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ option(INTERNAL_MODULE_PPM "Support for PPM internal module" OFF)
 option(AUTOUPDATE "Auto update internal chips from SD" OFF)
 option(BIND_KEY "Enable bind button" OFF)
 option(BLUETOOTH "FrSky BT module support" OFF)
+option(LUA_MIXER "Enable LUA mixer/model scripts support" ON)
```

Comment:
In case LUA_MIXER is enabled now for all targets (color and b/w), this line could be moved into main radio CMakeLists.txt, as otherwise it is present in the same way in all target CMakeLists.txt.

Reply:
Sounds good to me... X9D+2019 as the target that seems to have the least amount of free flash still has 3kb remaining, so it looks like we've clawed back enough flash to leave it on globally for now (that was the reason was only enabled for horus targets before - bw targets were failing when I enabled for all). If we need to revert this in the future, it can be exposed as a cloudbuild compile option instead.